frontend-v2/src/app/course/[id]/page.tsx

```diff
@@ -0,0 +1,149 @@
+import Image from "next/image";
+import waves from "../../../assets/waves.svg";
+import { LinkIcon } from "@heroicons/react/24/solid";
+import { Suspense } from "react";
+import TermsGroup from "@/components/TermsGroup";
+import Link from "next/link";
+import DoughnutChart from "@/components/DoughnutChart";
+import { notFound } from "next/navigation";
+import ReviewsBar from "../../../components/review/ReviewsBar";
+import Rating from "@/components/Rating";
+import ReviewSearchbar from "@/components/review/ReviewSearchBar";
+
+/**
+ * Fetch course from backend based on the given id
+ */
+const getCourse = async (id: string) => {
+  const response = await fetch(`http://localhost:3030/api/v1/courses`, {
```

Comment:
Vincents now working on a single course API to avoid this, this wouldn't of worked for 2 reasons:
1. It only returns the top 25 courses
2. Iterating over every course to find the one this page is meant for isn't the best idea for performance